### ISSUE TITLE
Design for Global sprint

### ISSUE THREAD

**Issue by xmatthewx** [MozillaFoundation/Design#254]
Global sprint is coming up. We have a few design tasks. They are likely...

 - Landing page on foundation site
 - ~~UX for Nav (on site, and github pages)~~
 - Graphic for social ~~, gh pages, etc~~
 - ~~UI for Header and nav on github pages~~

Brief is coming soon. cc @zee-moz 

Design can start after we assess tech requirements with @alanmoo to determine what will land on site vs gh pages.

**Comment by kristinashu** [comment]
Here's the brief: https://docs.google.com/document/d/144BpwxDh0oCwDAI1SrbWmtZyEoBuj11vivjhW5Z8_m4/edit

**Comment by kristinashu** [comment]
@zee-moz @xmatthewx we'll be chatting tomorrow but in the meantime I will start to:
- move content from https://mozilla.github.io/global-sprint/ over to a Foundation site CMS page
- create images and icons for the CMS pages, tito, teachable, and social media

**Comment by kristinashu** [comment]
I've also created an overview of where content could live https://redpen.io/zd25149b8c0dab27bf

**Comment by kristinashu** [comment]
Notes from our meeting:
- Roughly copied and pasted content from https://mozilla.github.io/global-sprint/ https://foundation.mozilla.org/opportunity/global-sprint/
- [Kristina] Style the content and add images
- [Kristina] Create images and icons for the CMS pages, tito, teachable, and social media
- [Matthew] Test map embed 
- [Zannah] Update 2017 content
  - reduce amount of copy
  - minor tweeks
  - move “The Global Sprint and Internet Health” on Participant page to the Home page
- [Zannah] Create and link to blog posts (success stories from last years sprint)
- [Kristina] Link to https://www.mozillapulse.org/search?keyword=global%20sprint
- [Kristina] Find a way to distinguish between 2017 vs 2018 projects in Pulse
- [Zannah] Tito: setup and links
- Phase 2: Market participation https://docs.google.com/document/d/1vuT1oR_xTQYRBewYbjKpfygnDUXl352BXZ2wrZhaeII/edit?usp=drive_web&ouid=113985443362969541279

**Comment by stephwright** [comment]
Hey all... loving what the site looks like so far!  Even got a comment from one of our partners that "the site looks slick!" 👍 

Couple thoughts/suggestions/requests:
1) We have one associated partner with this event already and are looking at options for sponsorships.  Part of our "benefit package" is display of their logo on the GS site.  Can we have a "partners" or "supporters" page for that?

2) I think the big yellow "sign up" button in the header may be a bit confusing for folks, particularly on the GS registration page.  Folks might think it's the "Sign Up" for the GS and it's not.  (True confession time: that's what I thought it was.)

3) also thinking we might need to be more specific in our tagging of Pulse projects… right now there are Pulse records coming up that are about the Global Sprint but aren’t necessarily projects… might be confusing / annoying to folks who are just looking for projects... maybe tagging projects specifically as GS projects somehow? rather than just blog posts ABOUT the sprint?

**Comment by xmatthewx** [comment]
> 2. I think the big yellow "sign up" button in the header...

We won't be able to change this now, though it is something we can consider as we iterate on some other site things this spring.

**Comment by kristinashu** [comment]
Good points @stephwright! Here are a few thoughts:
1. Roughly how many sponsors are you thinking? I would prefer to add them at the bottom on the home page rather than add a new dedicated page.
2. We could add a prominent "Register now" button CTA on the landing page to try and guide users there.
3. We're explore ideas for that. Maybe adding a banner to office projects or having a specific #globalsprint2018 tag...

**Comment by kristinashu** [comment]
Here's the design direction I'm thinking of. I'm going with a fun, DIY and maker style approach. 

![global-sprint-02](https://user-images.githubusercontent.com/1068783/36163859-85d4030e-109f-11e8-9ff9-d6da94dbe837.jpg)

**Comment by kristinashu** [comment]
@zee-moz do you have access to the CMS now? Are you ok updating the copy on [the site](https://foundation.mozilla.org/opportunity/global-sprint/) based on @tlasadea [feedback](https://docs.google.com/document/d/1hQFYKynOhPv1wFDNUaATZT1M_wFeXAIZKnlmkUN6lk0/edit?usp=sharing)?

I will continue to make graphics for the site and social media.

**Comment by zee-moz** [comment]
@kristinashu thanks for making graphics, yay! @acabunoc @Saallen  and @stephwright (who is out sick today) pls take a look at this design direction!

**Comment by kristinashu** [comment]
@zee-moz has updated all the copy on the site and I've started styling it https://foundation.mozilla.org/opportunity/global-sprint/

@tlasadea please make any additional copy edits in the CMS. Note, styling in the CMS is finicky and bullet lists aren't working but there's a ticket to fix it.

**Comment by alanmoo** [comment]
PR has been submitted for the bullet issue, so style them as bullets in the WYSIWYG field and they'll show up properly on the published page once the fix lands.

**Comment by zee-moz** [comment]
@kristinashu some feedback from me on the graphics, now that I've got that content up on the CMS...  I love the sketchy work-in-process look and the retro colors. The one thing that I think is missing is some stronger indication that code or data is involved-- right now I think it's leaning heavily towards craft/by hand/physical stuff. Which does happen at the sprint, but I think most of the work is still digital. Thanks again for your help on this :)

**Comment by kristinashu** [comment]
@zee-moz sure, I can add some hints of code and data.

I've also opened a separate ticket for image assets external to the site https://github.com/MozillaFoundation/Design/issues/263

@zee-moz can you please confirm what we need to launch the initial phase of the site?
- [Landing page](https://foundation.mozilla.org/opportunity/global-sprint/):
  - Update Registration link to a 'Signup for Email Notification' link
  - Remove links to 2017 Stories until we have them
- [Register](https://foundation.mozilla.org/opportunity/global-sprint/register/):
  - Hide this page until we have hosts?
- [2018 Projects](https://foundation.mozilla.org/opportunity/global-sprint/2018-projects/):
  - Hide this page until we have project leads?
- [For Participants](https://foundation.mozilla.org/opportunity/global-sprint/for-participants/):
  - Add a link to Signup for Email Notification
- [For Project Leads](https://foundation.mozilla.org/opportunity/global-sprint/for-project-leads/):
  - Add link to 2017 Project examples
  - Add link to guide
- [For Site Hosts](https://foundation.mozilla.org/opportunity/global-sprint/for-site-hosts/):
  - Add link to guide

**Comment by zee-moz** [comment]
@kristinashu this looks about right.
- I imagine we could just create a generic form to sign up for interest-- rather than creating a Ti.To site for general interest
- I should have final 2017 stories by midday tomorrow! Profiles were created, getting participant reviews and images now.
- I am in the process of updating and transferring the guides. 
- There are a few small things to do to the Project and Site registration forms, but those links should be done tomorrow as well. 

I'm thinking we'll launch/announce Tues/Weds week, so we don't get lost in the long weekend email pile-up. Also that will give folks time to review content and stories. 
 @KevZawacki @Saallen @stephwright  FYI

**Comment by Saallen** [comment]
@ Zannah- I think for generic sign up for interest would be better through ti-ti as you then have one home of all emails, and can access them as one bulk group.
Think Tuesday for first announcements is great

**Comment by KevZawacki** [comment]
@zee-moz Wednesday, 2/21 would be best. We're announcing the WINS winners on Tuesday, 2/20, so the Sprint post might be lost in the shuffle.

**Comment by stephwright** [comment]
Sorry folks... finally back among the land of the living.  @kristinashu all your suggestions sound great to me.  Not sure on the number of sponsors yet.  I can't imagine we'll have a ton... 3-5?

**Comment by zee-moz** [comment]
@KevZawacki  @tlasadea
here the 2017 "sprint stories." I took interview notes and crafted short narratives. @KevZawacki  @tlasadea Pls review. @kristinashu will the profile photos included here work? We might need a better profile photo of Arliss (also need more TOR  sprint photos).  

- [Geraldo, Participant & Site Host](https://docs.google.com/document/d/1XzHdZ1vdyS-uXUt5a2kfEigVYy7z_iJ7-weJIBpMUqs/edit)
- [Daniela & Samantha, Project Co-Leads](https://docs.google.com/document/d/16oUrEeIqapyLLCEpNE5SlahCXVNWSy7KBg1YC2cwbI4/edit)
- [Arliss, Site Host](https://docs.google.com/document/d/1Echk27MGrLNi-4bBMa9gMEFRQkiD6_bHB4ZxPKdJCQ4/edit)

I may have one other story from a site host in India (could swap for Arliss, for more international flair), but I'm not sure when I'll get the content and assets.  

Finally-- these will be hosted on a blog and linked to the site-- @KevZawacki which of our blogs is the best spot for the posts?

**Comment by KevZawacki** [comment]
Hi @zee-moz. Let's publish on medium.com/read-write-participate. Just let me know when you'd like me to post.

**Comment by kristinashu** [comment]
@zee-moz can you put the photos in the [drive folder](https://drive.google.com/open?id=1GVXBThqetqucquNaQ6Yb5U0q32LvldXM)? I think google docs might compresses the images.

**Comment by zee-moz** [comment]
@KevZawacki these stories are ready to post on RWP

- [Geraldo, Participant & Site Host](https://docs.google.com/document/d/1XzHdZ1vdyS-uXUt5a2kfEigVYy7z_iJ7-weJIBpMUqs/edit)
- [Daniela & Samantha, Project Co-Leads](https://docs.google.com/document/d/16oUrEeIqapyLLCEpNE5SlahCXVNWSy7KBg1YC2cwbI4/edit)
- [Arliss, Site Host](https://docs.google.com/document/d/1Echk27MGrLNi-4bBMa9gMEFRQkiD6_bHB4ZxPKdJCQ4/edit)

we have images on the foundation site for each, ready to link out. Let me know when you post them-- they are one of the very few things left to do before launch.

**Comment by KevZawacki** [comment]
Thanks, @zee-moz.

@kristinashu mentioned she'd be publishing. @kristinashu, do you need a hand?

**Comment by kristinashu** [comment]
I got it Kevin! I'd like to resize some of the images as I do it and then update the link on the site.

For the site: we will be adding/updating a few links then it should be ready to go! https://foundation.mozilla.org/opportunity/global-sprint/

Note: Register and 2018 Projects menu items are hidden until we have Hosts and Projects.

All images for social media and ti.to are here https://drive.google.com/open?id=1B3FRBv6Xt_uZAEPqaEsJWoib5Rl3pOwc

Couple follow up tickets: 'Add partner logos to Global Sprint site #264' and 'Add banner tag to all Global Sprint 2018 projects in Pulse #265'

**Comment by KevZawacki** [comment]
Thanks, @kristinashu.

@zee-moz, just send me a note when I should publish the blog. Thanks!
